config/wbstack.php

```diff
@@ -21,6 +21,8 @@
 
     'elasticsearch_hosts' => array_filter(explode(',', env('ELASTICSEARCH_HOST', ''))),
     'elasticsearch_enabled_by_default' => env('WBSTACK_ELASTICSEARCH_ENABLED_BY_DEFAULT', false),
+    'elasticsearch_cluster_to_index' => env('ELASTICSEARCH_CLUSTER_TO_INDEX', null),
```

Comment:
Would this be better named cluster to create shared index on? Since this is explicitly not used by alias index creation code?